docs/source/ops/monitoring.rst

```diff
@@ -0,0 +1,130 @@
+Monitoring
+==========
+
+This section gives a high-level overview of the PrivateStorageIO monitoring efforts.

```

Comment:
The content here generally looks good to me.  I like the references to work by people with more experience than me.  I wonder how further documentation will fit in with this high-level information - but I guess I'll find out. :)

I have one one extremely trivial point about the stylization of the project name.  he name of the company is "PrivateStorage.io, LLC.".  The name of this ops software project is "PrivateStorageio".  There is another ops project that deals with the surrounding/supporting infrastructure called "PrivateStorageOps".  The domain name used for the website and other network services associated with this deployment is "privatestorage.io" (but may change to "private.storage" at some point).  Perhaps it makes sense to refer to the domain name here?  Alternatively, if we think the monitoring config is going to apply generally to any deployment of the software then maybe referring to this infrastructure project, "PrivateStorageio", makes sense.  I think I like this (though maybe it also suggests this project is misnamed since we're using it for, eg, the HRO Cloud deployment too).

Reply:
I took the liberty of putting much of your last paragraph in an extra section of the docs.